store/unit/store.ts

```diff
@@ -36,7 +36,7 @@ export const useUnitStore = defineStore('unitStore', () => {
   }
 
   function setActiveUnitIdBySlug(unitSlug: Unit['slug']) {
-    activeUnitId.value = unitBySlug(unitSlug).value?.id;
+    activeUnitId.value = unitBySlug(unitSlug).value?.id || '';
```

Comment:
Is right to assign an empty value to the `activeUnitId`? Or just avoid to change the value?

```suggestion
    const unit = unitBySlug(unitSlug);
    if (unit.value) {
      activeUnitId.value = unit.value.id;
    }
```

Reply:
I think it's right assign an empty value. `''` is the default value and if we have a unit for which we don't find the `id`, I think we should have `''` instead of the previous value.
I can be wrong, so let's ask @Tansito and @korgan00 , just in case I'm missing something

Reply:
I understand why we are using the **default value** but for me it's weird that the default value is an **empty string**. I don't know if there is a reason behind but I would say `null` or `undefined` would be better values.

Reply:
I fine with extending type to null or undefined but I don't have strong opinions here.

Reply:
Done https://github.com/Qiskit/saiba/pull/324/commits/714bd0c044137c2f4e3771301e923a32ec7211df